open-api.yaml

```diff
@@ -1476,17 +1503,31 @@ paths:
                     type: string
                     example: '["title", "overview"]'
                     default: '[]'
+                highlightPreTag:
+                  type: string
+                  description: Specify the tag to put before the highlighted query terms.
+                  example: '<mark>'
+                  default: '<em>'
+                highlightPostTag:
+                  type: string
+                  description: Specify the tag to put after the highlighted query terms.
+                  example: '</mark>'
+                  default: '</em>'
                 attributesToCrop:
                   type: array
                   description: Array of attributes whose values have to be cropped. Cropped attributes are returned in `_formatted` response object.
                   items:
                     type: string
                     example: '["overview", "author"]'
                     default: '[]'
+                cropMarker:
+                  type: string
+                  description: Sets the crop marker to apply before and/or after cropped part selected within an attribute defined in `attributesToCrop` parameter.
+                  default: '…'
                 cropLength:
                   type: number
-                  description: Length used to crop field values.
-                  default: 200
+                  description: Sets the total number of **words** to keep around the matched part of an attribute specified in the `attributesToCrop` parameter.
```

Comment:
By reading this, I understand that we will have XX words before and XX words after the match, not that the number of words returned will be XX.
Example : 
- Consider this phrase: `In his ravenous hatred he found no peace`
- Set `cropLength` to 2.
- I search for `hatred`

This description makes me think I will receive `his ravenous hatred he found` because I will have 2 words before and after `hatred` , which is in contradiction with what I've read below in this PR.